Tests/Data/HydroMechanics/StaggeredScheme/InjectionProduction1D/InjectionProduction1D.prj

```diff
@@ -48,6 +43,16 @@
                             <type>Constant</type>
                             <value>1e-3</value>
                         </property>
+                        <property>
+                            <name>density</name>
+                            <type>Linear</type>
+                            <reference_value>1e4</reference_value>
+                                <independent_variable>
+                                    <variable_name>phase_pressure</variable_name>
+                                    <reference_condition>0</reference_condition>
+                                    <slope>3.3332941333333e-5</slope>
```

Comment:
Why is the slope different from the fluid's compressibility?

Reply:
The compressebility of 3.333... before probably was misscalculated from my side but did not seem to influence the result enough to fail. This value now should correctly result in a storage of 1e-5.